test/integration/discovery_test.go

```diff
@@ -45,6 +45,11 @@ func TestDiscoveryGroupVersions(t *testing.T) {
 	expectedGroupVersions := sets.NewString()
 	for group, versions := range configapi.KubeAPIGroupsToAllowedVersions {
 		for _, version := range versions {
+			// TODO: The policy is enabled by kube but does not have client registered
```

Comment:
revert this... the failure here means the API group is not enabled... either remove the group from KubeAPIGroupsToAllowedVersions or actually register/enable it